addons/web_editor/static/src/js/common/utils.js

```diff
@@ -279,7 +278,24 @@ function _colorHEXtoHSL(hex) {
 
     return {h, s, l};
 }
-
+/**
+ * Convert HSL color to HEX Color.
+ *
+ * @param {number} h hue
+ * @param {number} s saturation
+ * @param {number} l lightness
+ * @returns {String} hex color.
+ */
+function _colorHSLToHEX(h, s, l) {
+    l /= 100;
+    const a = s * Math.min(l, 1 - l) / 100;
+    const f = n => {
+        const k = (n + h / 30) % 12;
+        const color = l - a * Math.max(Math.min(k - 3, 9 - k, 1), -1);
+        return Math.round(255 * color).toString(16).padStart(2, '0'); // convert to Hex and prefix "0" if needed
+    };
+    return `#${f(0)}${f(8)}${f(4)}`;
+}
```

Comment:
Same: already something which is defined in colorpicker.js (from rgba, but it can be converted first).